src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java

```diff
@@ -59,6 +59,7 @@
   private static boolean testStarted = false;
   private By lastFindBy;
   private WebDriver driver;
+  public static Throwable lastThrow = null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make lastThrow a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this "public static lastThrow" field final [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1444)